packages/core/src/components/sections/RegionBar/RegionBar.tsx

```diff
@@ -30,10 +31,14 @@ type RegionBarSectionProps = {
 }
 
 function RegionBarSection({ ...otherProps }: RegionBarSectionProps) {
+  const { isDesktop } = useScreenResize()
+
   return (
-    <Section className={`${styles.section} section-region-bar display-mobile`}>
-      <RegionBar {...otherProps} />
-    </Section>
+    !isDesktop && (
```

Comment:
Previously, I made this change to address a CLS issue. But, I'm reverting it now, using `isDesktop` prevents `RegionBar` and `RegionButton` from being rendered simultaneously (which was causing a ref issue :/). I’ve checked again and haven’t noticed any CLS issues since.